addon/components/document-details.js

```diff
@@ -41,4 +44,25 @@ export default class DocumentDetailsComponent extends DocumentCard {
       this.notification.danger(this.intl.t("alexandria.errors.update"));
     }
   }
+
+  @dropTask *uploadReplacement(event) {
+    const [file] = event.target.files;
```

Comment:
Please wrap this in a `try ... catch` like the saveDocument or deleteAndReset and issue the appropriate notifications

Reply:
Okey

Reply:
Done